### PR TITLE
Changed plot title

### LINKED ISSUE #52: Plot titles need style refining
They are too long. Perhaps we could just show in title style the key words, and add a line under the plot that has the currently proposed title.

### LINKED ISSUE #53: "plot not found" alt message should use `makeTitle`

### PR DESCRIPTION
**Pull request by acholyn**
made alt use `makeHr`, closes https://github.com/UCL/UKCORDEX-plot-explorer/issues/53
Closes https://github.com/UCL/UKCORDEX-plot-explorer/issues/52

### CODE REVIEW

**Comment by raquelalegre** [comment]
Thanks for this! It'll help the discussion with Clair today. I was wondering if we could add some padding to the border so it's not right next to the first letter of the caption.

**Comment by raquelalegre** [comment]
We can leave this PR open until we get feedback later today.

File: src/components/FigureRow.js [hunk 1]
```diff
@@ -30,13 +30,19 @@ function FigureRow({ plotvars, seasons, periods, plottypes, images, regions }) {
...
+                  {makeHr(plot.slice(plot.lastIndexOf("/") + 1))}
+                </h4>
                 <Figure.Caption>
                   {makeTitle(plot.slice(plot.lastIndexOf("/") + 1))}
                 </Figure.Caption>
                 <Image
                   fluid="true"
                   // width="100%"
-                  alt={"plot not found"}
+                  alt={`plot ${makeTitle(
```
Comment (raquelalegre) [inline src/components/FigureRow.js]: This is hard to test but it looks good!

**Comment by acholyn** [comment]
Closes https://github.com/UCL/UKCORDEX-plot-explorer/issues/52

**Review by raquelalegre — APPROVED** [review]

**Pull request merged**